package/yast2.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jul 22 16:08:16 UTC 2020 - David Diaz <dgonzalez@suse.com>
+
+- Add a CFA based class to manage nsswitch.conf (bsc#1173119).
```

Comment:
well, I also expect that it will be used. So I would like to see also its usage.

Reply:
Yes, the idea is

* to adapt [yast-pam/Nsswitch.rb](https://github.com/yast/yast-pam/blob/96a24a906231730c90bca79d3754da9cc4e70b1c/src/modules/Nsswitch.rb) module to use it instead the SCR agent, or
* to directly use it replacing the [Nsswitch.ReadDb](https://github.com/search?q=%22Nsswitch.ReadDb%22+user%3Ayast&type=Code) and [Nsswitch.WriteDb](https://github.com/search?q=%22Nsswitch.WriteDb%22+user%3Ayast&type=Code) calls. 

What do you think would be better?

Reply:
As discussed in IRC, the first option is _the right one_ since 

> \<jreidinger> dgdavid: and note that we cannot replace all usage of NSSSwitch module as it is used also in samba, which is in perl.